web/package.json

```diff
@@ -60,6 +60,7 @@
     "react-feather": "^2.0.3",
     "react-hot-loader": "^4.12.14",
     "react-linkify": "^1.0.0-alpha",
+    "react-markdown": "^4.3.1",
```

Comment:
This package isn't used elsewhere in the PR, I think you forgot to remove it

Reply:
My bad, it was for the initial commit to show markdown on web version